report_thesis/src/sections/proposed_approach/model_selection.tex

```diff
@@ -34,15 +34,9 @@ \subsection{Model and Preprocessing Selection}
 
 To further bolster our selection pool, we included models from the same family as those that showed promise in the literature.
 
-\gls{xgboost} was included in our experiments based on its promising accuracy in various settings.
-For example, \citet{xgboost_in_biomedicie} showed that \gls{xgboost} outperformed models such as \gls{rf} and \gls{svm} in predicting biological activity based on quantitative description of the compound's molecular structure. 
-Another example is \citet{xgboost_in_heart_disease}, who used \gls{xgboost} to predict heart disease and found that \gls{xgboost} outperformed \gls{rf} and \gls{etr} in correctly classifying patients with heart disease.
-Due to these factors and the limited study of \gls{xgboost} in the context of \gls{libs} data, we decided to include it in our experiments.
-
-Following the same logic, we included \gls{ngboost}. 
-\gls{ngboost} is a recent model introduced by \citet{duan_ngboost_2020} that, according to their work, improves upon \gls{gbr} by using a more sophisticated loss function and a more advanced gradient boosting algorithm.
-Limited research has been conducted using this algorithm in the context of \gls{libs} data. 
-However, \citet{ngboost_landslide} showed that \gls{ngboost} outperformed \gls{xgboost} and \gls{rf} in correctly determining landslide-prone fields, with an AUC of 0.898 compared to 0.871 and 0.863 for \gls{xgboost} and \gls{rf}, respectively.
+\gls{xgboost} and \gls{ngboost} both belong to the gradient boosting family, but they approach gradient boosting in distinct ways.
+\gls{xgboost} uses a more advanced gradient boosting algorithm, while \gls{ngboost} uses a more sophisticated loss function.
```

Comment:
```suggestion
\gls{xgboost} uses advanced algorithmic optimizations, while \gls{ngboost} focuses on sophisticated probabilistic loss functions.
```

Reply:
We could say more:
```suggestion
\gls{xgboost} uses advanced algorithmic optimizations, such as regularization, tree pruning, and parallel processing, to improve performance and prevent overfitting. 
On the other hand, \gls{ngboost} focuses on sophisticated probabilistic loss functions, optimizing the natural gradient to model the entire probability distribution of the target variable, making it well-suited for tasks requiring uncertainty estimation and probabilistic forecasting.
```

I'm sure about XGBoost, but can @Pattrigue confirm the NGBoost statement?

Reply:
My suggested changes are mostly about granularity. We want to say _something_, because there's more reason than simply trying a variation of the same algorithm.